dmn-engine/src/test/scala/org/camunda/dmn/AuditLogTest.scala

```diff
@@ -47,6 +53,69 @@ class AuditLogTest extends AnyFlatSpec with Matchers with DecisionTest {
     auditLog.requiredEntries.size should be(0)
   }
 
+  it should "contains the result of a decision table if there is none" in {
+
+    eval(discountDecision,
+         "discount",
+         Map("customer" -> "Other", "orderSize" -> 7))
+
+    auditLog.rootEntry.id should be("discount")
+    auditLog.rootEntry.name should be("Discount")
+    auditLog.rootEntry.decisionLogic shouldBe a[ParsedDecisionTable]
+    auditLog.rootEntry.result shouldBe a[DecisionTableEvaluationResult]
+
+    val result =
+      auditLog.rootEntry.result.asInstanceOf[DecisionTableEvaluationResult]
+    result.inputs.size should be(2)
+
+    result.inputs(0).input.id should be("input1")
+    result.inputs(0).input.name should be("Customer")
+    result.inputs(0).value should be(ValString("Other"))
+
+    result.inputs(1).input.name should be("Order Size")
+    result.inputs(1).value should be(ValNumber(7))
+
+    result.matchedRules.size should be(0)
+
+    result.result should be(ValNull)
+
+    auditLog.requiredEntries.size should be(0)
+  }
+
+  it should "contains the result of a decision table if there is an exception" in {
```

Comment:
Make the description a bit more clear.

```suggestion
  it should "contains the result of a decision table if a failure occurred" in {
```